javascript-local-storage/data.js

```diff
@@ -0,0 +1,8 @@
+/* exported todos */
+
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+var todos = JSON.parse(previousTodosJSON);
```

Comment:
You are missing setting `todos` equal to an empty array to start, and then you should replace it _**if**_ there is a value stored in localStorage